doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3227,6 +3227,22 @@ NCSDK-28192: Hardfault in :c:func:`coap_codec_agnss_encode`
 
   ** Workaround:** Apply the fix from `sdk-nrf PR #16242`_.
 
+.. rst-class:: v2-9-0-nRF54H20-1 v2-9-0 v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
+
+NCSDK-31645: Download client might fail with error ``-EMSGSIZE`` when using TLS on nRF91 Series devices
+  The nRF91 Series modem has a size limit of 2kB for receiving TLS packages.
```

Comment:
I think it was better before when you wrote the explicit amount of bytes (2303). Why the change?

Reply:
Because 2kB is what is documented in modem firmware release.